tests/integration/CMakeLists.txt

```diff
@@ -66,7 +66,7 @@ blt_add_executable(
 
 target_include_directories(
   allocator_accessibility_tests
-  PRIVATE
+  PUBLIC
```

Comment:
I think this in unnecessary.

Reply:
Ah, hmm. This might have been left over from experimenting. I'll revert it. Thanks!